src/cli/exec_command.zig

```diff
@@ -0,0 +1,42 @@
+const bun = @import("root").bun;
+const string = bun.string;
+const Output = bun.Output;
+const Global = bun.Global;
+const Environment = bun.Environment;
+const strings = bun.strings;
+const MutableString = bun.MutableString;
+const stringZ = bun.stringZ;
+const default_allocator = bun.default_allocator;
+const C = bun.C;
+const std = @import("std");
+const open = @import("../open.zig");
+const Command = bun.CLI.Command;
+
+pub const ExecCommand = struct {
+    const discord_url = "https://bun.sh/discord";
+    pub fn exec(ctx: Command.Context) !void {
+        const script = ctx.positionals[1];
+        // this is a hack: make dummy bundler so we can use its `.runEnvLoader()` function to populate environment variables probably should split out the functionality
+        var bundle = try bun.Bundler.init(
+            ctx.allocator,
+            ctx.log,
+            try @import("../bun.js/config.zig").configureTransformOptionsForBunVM(ctx.allocator, ctx.args),
+            null,
+        );
+        try bundle.runEnvLoader(false);
+        const mini = bun.JSC.MiniEventLoop.initGlobal(bundle.env);
+        const code = bun.shell.Interpreter.initAndRunFromSource(mini, "<script>", script) catch |err| {
```

Comment:
Instead of `<script>`, it should be the absolute file path of the current working directory with `/[eval]` appended at the end, to match `bun -e`